bases/mundo_bm_wdi/country/table_config.yaml

```diff
@@ -0,0 +1,510 @@
+      description: Special notes regarding currency information and if it is a fiscal
+          year or calendar year
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: income_group
+      bigquery_type: string
+      description: Determines the income group the country belongs
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: national_accounts_base_year
+      bigquery_type: string
+      description: National accounts base year
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: national_accounts_reference_year
+      bigquery_type: float
+      description: National accounts reference year
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: sna_price_valuation
+      bigquery_type: string
+      description: System of national accounts price evaluation
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: lending_category
+      bigquery_type: string
+      description: Define types of lending category
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: other_groups
+      bigquery_type: string
+      description: Other abbreviations
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: system_of_national_accounts
+      bigquery_type: string
```

Comment:
se for ano a variável, o tipo é inteiro

Reply:
em system_of_national_accounts a informação vem como texto. ex.: Country uses the 1993 System of National Accounts methodology. acredito que dê pra usar uma regex pra obter somente o ano dessa frase, o que acha?